src/Samples/AuthoringDemo/AuthoringDemo/AuthoringDemo.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+        <Platforms>x64</Platforms>
+        <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    </PropertyGroup>
+
+    <!-- CsWinRT properties -->
+    <PropertyGroup>
+        <CsWinRTComponent>true</CsWinRTComponent>
+        <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>
+        <CsWinRTEnableLogging>true</CsWinRTEnableLogging>
+        <GeneratedFilesDir Condition="'$(GeneratedFilesDir)'==''">$([MSBuild]::NormalizeDirectory('$(MSBuildProjectDirectory)', '$(IntermediateOutputPath)', 'Generated Files'))</GeneratedFilesDir>
+    </PropertyGroup>
+    
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.1.1" />
+    </ItemGroup>
+
+    <!-- NuGet package generation properties -->
+    <PropertyGroup>
+        <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+        <Version>1.0.0</Version>
+        <PackageOutputPath>.\nuget</PackageOutputPath>
+    </PropertyGroup>
+    
+    <ItemGroup>
+        <Content Include="AuthoringDemo.targets" PackagePath="build;buildTransitive" />
```

Comment:
You might need to add the `Pack` metadata value on this... Like I do [here](https://github.com/microsoft/CsWinRT/blob/cb416d230470469fd936e6a785701ef8c9ee022a/nuget/Microsoft.Windows.CsWinRT.targets#L197)

Reply:
Seems to work for me without `Pack` - I see both targets file in build and buildTransitive